conan/cli/command.py

```diff
@@ -98,6 +98,15 @@ def _format(self, parser, info, *args):
 
         formatter(info)
 
+    def _raise_errors(self, info):
+        if info and type(info) == dict:
+            if "conan_error" in info and isinstance(info["conan_error"], Exception):
+                raise info["conan_error"]
+            if "conan_warning" in info and isinstance(info["conan_warning"], Exception):
+                try:
+                    raise info["conan_warning"]
+                except ConanException as e:
+                    ConanOutput().warning(e)
```

Comment:
This expression is difficult to understand, I'd expect to just print the warnings.